src/main/java/io/reactivex/observers/BaseTestConsumer.java

```diff
@@ -403,6 +403,31 @@ public final U assertNever(Predicate<? super T> valuePredicate) {
 
     /**
      * Asserts that this TestObserver/TestSubscriber received an onNext value at the given index
+     * which is equal to the given value with respect to Objects.equals.
```

Comment:
`Objects` is a Java 8 component but `ObjectHelper` is internal, thus maybe let's say "with respect to null-safe Object.equals()".